test/functional/wallet_reorgsrestore.py

```diff
@@ -72,6 +72,8 @@ def run_test(self):
         self.sync_blocks([self.nodes[0], self.nodes[2]])
         conflicted = self.nodes[0].gettransaction(conflicted_txid)
         conflicting = self.nodes[0].gettransaction(conflicting_txid)
+        self.nodes[0].syncwithvalidationinterfacequeue()
+        self.nodes[2].syncwithvalidationinterfacequeue()
         assert_equal(conflicted["confirmations"], -9)
         assert_equal(conflicted["walletconflicts"][0], conflicting["txid"])
```

Comment:
These changes aren't doing anything. The sync call is being done after obtaining the data from the node.